### ISSUE TITLE
example urls / links

### ISSUE THREAD

**Issue by jbenet** [neocities/hshca#1]
would be nice to show some example urls + links in the readme somewhere so reader can see exactly how to go from a content multihash --> a hshca url..

something like:

```
ipfs: /ipfs/QmYNmQKp6SuaVrpgWRsPTgCQCnpxUYGq76YEKBXuj2N4H6/foo
basic gateway:  http://ipfs.io/ipfs/QmYNmQKp6SuaVrpgWRsPTgCQCnpxUYGq76YEKBXuj2N4H6/foo
hshca:  http://<encoded thing>.neocities.org/foo
```

or something like this?

**Comment by donpdonp** [comment]
Exactly. the "example" section is actually an example implementation. I'm trying to grasp how the content hash is used in the URL.

**Comment by kyledrake** [comment]
Agreed. Thanks for the feedback! I'll try to get to this tonight.
